src/components/GradeLevelFilter.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Button, Popover } from '@material-ui/core';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+const useStyles = makeStyles((theme) => ({
+  buttonStyle: {
+    margin: theme.spacing(1),
+    height: theme.spacing(4),
+  },
+  popOverWindow: {
+    minWidth: '200px',
+    fontSize: '15px',
+    paddingTop: theme.spacing(1),
+    paddingLeft: theme.spacing(1),
+    paddingBottom: theme.spacing(1),
+  },
+}));
+
+const gradeItems = [
+  '12th - High School',
+  'College Freshman',
+  'College Sophomore',
+  'College Junior',
+  'College Senior',
+  '5th Year',
+  'Post Grad',
+];
+
+export default function GradeLevelFilter() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState(null);
+  const handleClick = (event) => setAnchorEl(event.currentTarget);
```

Comment:
`handleClick` is a bit generic. I can't guess based on the name what it's supposed to do. Maybe `openPopover`?

Reply:
Ok `openPopover` it is